.github/workflows/scanner-versioned-definitions-update.yaml

```
@@ -159,12 +158,24 @@ jobs:
     - name: Create bundle output directory
       run: mkdir -p definitions/${{ env.SCANNER_BUNDLE_VERSION }}
 
+    - name: Sanity check NVD zip
+      run: |
+        if [ ! -f ./nvd.zip ]; then
+          echo >&2 "nvd.zip not found!"
+          exit 1
+        fi
```

One last thing. Since this is to overcome a limitation of the updater, can we extend this check to be:

```suggestion
        echo "STACKROX_NVD_ZIP_PATH=$PWD/nvd.zip" >> $GITHUB_ENV
        echo "checking contents of $STACKROX_NVD_ZIP_PATH"
        unzip -l "$STACKROX_NVD_ZIP_PATH"
```

Then we don't need the `export` calls anymore, and file is guarantee to both exist and be a valid zip.

Some ref if you want to learn more: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#environment-files

---

BTW, this is how `unzip` fails if the file does not exist:

```
$ unzip -l foobar                                           [~/wt/use-versioned-vulns/github.com/stackrox/stackrox]
unzip:  cannot find or open foobar, foobar.zip or foobar.ZIP.
```

So, the workflow output would be something like:

```
checking contents of /github/workspace/nvd.zip"
unzip:  cannot find or open /github/workspace/nvd.zip, /github/workspace/nvd.zip.zip or /github/workspace/nvd.zip.ZIP.
```

And on success it would print the contents of `nvd.zip`, which is kinda nice.

---

Thanks, this is much better than the original sanity check although I haven't re-run the bundle update jobs yet.

---

Unresolving because I am seeing some additional changes are still needed to fully onboard this (and I believe your plan is to onboard it).

---

Good now.